### PR TITLE
LPS-127391 Hide controls for detail view

### PR DESCRIPTION
**Pull request by jotascript**
cc: @interaminense

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:stop

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ViewEntryUpperToolbar.es.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ function ViewEntryUpperToolbar({
 	additionalButtons,
 	children,
 	dataRecordId,
```
Comment (rafaprax) [inline modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ViewEntryUpperToolbar.es.js:30]: Hey @jotascript, please rename this to `showNavigationControls` or something like this.

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: portal-kernel/src/com/liferay/portal/kernel/workflow/packageinfo [hunk 2]
```diff
@@ -1 +1 @@
-version 9.6.0
\ No newline at end of file
+version 10.0.0
```
[the remark this answers is not in the document]
Comment (MarinhoFeliphe) [reply portal-kernel/src/com/liferay/portal/kernel/workflow/packageinfo:1]: Ok

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Comment by MarinhoFeliphe** [comment]
resent here [#279](https://github.com/liferay-workflow/liferay-portal/pull/279)

**Pull request closed without merging**